tests/end_to_end/test_op.py

```diff
@@ -97,3 +100,30 @@ def test_binops(execute):
     assert res.values["r10"] == 11
     assert res.values["r11"] == 9
     assert res.values["r12"] == 2
+
+
```

Comment:
want to see test case for these operations:
```(x := [t for t in range(3)])```
```x = 1
(x,y:=(1,2))
```

Reply:
also this:
```
import pandas
if ( lib := sys.modules['pandas']):
    print(lib.__version__)
```

Reply:
and finally one inside list comp:
```
import math
[res for x in range(10) if (res:= math.sin(x)) >= 0]
```

Reply:
added all tests